src/Core/src/Core/IMenuElement.cs

```diff
@@ -1,14 +1,16 @@
-﻿namespace Microsoft.Maui
+﻿using System.Collections.Generic;
+
+namespace Microsoft.Maui
 {
 	public interface IMenuElement : IElement, IImageSourcePart, IText
 	{
 		/// <summary>
 		/// Represents a shortcut key for a MenuItem.
 		/// </summary>
-#if NETCOREAPP || NETSTANDARD2_1_OR_GREATER
-		IAccelerator? Accelerator => null;
+#if NETSTANDARD2_0
+		IReadOnlyList<IAccelerator>? Accelerators { get; }
 #else
-		IAccelerator? Accelerator { get; }
+		IReadOnlyList<IAccelerator>? Accelerators => null;
 #endif
```

Comment:
Do we want to have a null collection, or an empty collection?

Null was fine when there was Yes or No, but now we have a collection so we can now use empty or not empty?

Reply:
I like empty / not empty

Reply:
will use null based on Shane's feedback in other thread